src/sentry/db/models/base.py

```diff
@@ -224,11 +224,6 @@ def __call__(self, model_class: Any) -> type:
         return model_class
 
 
-control_silo_model = ModelSiloLimit(SiloMode.CONTROL, read_only=SiloMode.REGION)
-region_silo_model = ModelSiloLimit(SiloMode.REGION)
-
-# Tags a model that is readable and writable in all silos. This should be used only
-# for data that is private to this particular Django stack and is not of concern to
-# the rest of the platform. This is experimental and may be reverted as we proceed
-# with work on the multi-region architecture.
-all_silo_model = ModelSiloLimit(*SiloMode)
+control_silo_with_replication_model = ModelSiloLimit(SiloMode.CONTROL, read_only=SiloMode.REGION)
+control_silo_only_model = ModelSiloLimit(SiloMode.CONTROL)
+region_silo_only_model = ModelSiloLimit(SiloMode.REGION)
```

Comment:
If these global decorator aliases become more trouble than they're worth, I wouldn't be opposed to ditching them and just referring to `ModelSiloLimit` directly, as in

```python
@ModelSiloLimit(SiloMode.CONTROL, read_only=SiloMode.REGION)
class TimeSeriesSnapshot(Model):
```

I'm not going to advocate for that yet because "control silo with replication model" is a useful name that conveys information. But, if find ourselves adding more cases with increasingly verbose names, I wouldn't want to see it sprawl into an unmanageable menagerie of decorators. In that case, I'd think it's important to let the init parameters tell the whole story and refactor until they make sense.

Reply:
That's fair.  Hopefully this is the final story for our data split, but if it does grow any more complex, I agree that we can just let the parameterization speak for itself.